CHALLENGE.md

```diff
@@ -94,9 +95,35 @@ You are given a prepared SPA project that was bootstrapped with Create React App
 ![create-tournament-prompt](.github/create-tournament-prompt.png)
 
 - Clicking on the `CREATE TOURNAMENT` button will open a browser prompt with a message `Tournament Name:`, an input field to enter the tournament name and the buttons `Cancel` to cancel and `OK` to confirm.
-- When confirming, the tournament will be created on the fake REST API and added on the UI without any loading indicators.
+- The tournament name must contain only Latin letters, numbers, and spaces, not an empty string or only spaces.
+- When confirming, the tournament will be created immediately using an optimistic update in the UI and a fetch call on the fake REST API.
+
+### 6. Make it responsive
+
+- The app must be responsive. Here you can find examples of how it should look like on different screen sizes:
+
+<details>
+  <summary>iPhone 12 Pro Max (428x926)</summary>
+  <img alt="iPhone 12 Pro Max" src=".github/iPhone-12-Pro-Max.png">
+</details>
+
+<details>
+  <summary>iPad (768x1024)</summary>
+  <img alt="iPad" src=".github/iPad.png">
+</details>
+
+<details>
+  <summary>Laptop 13" (1280x800)</summary>
+  <img alt="Laptop 13"" src=".github/Laptop-S.png">
+</details>
+
+<details>
+  <summary>iMac Retina 27" (2560x1440)</summary>
+  <img alt="iMac Retina 27"" src=".github/iMac-Retina-27.png">
+</details>
 
 ## Finished?
 
-- Create a `ZIP` file with your solution
-- Send it to the recruiter that sent you this challenge
+- Create a `ZIP` file with your solution.
+- Include the .git folder to the `ZIP` file to make it easier for us to review your code.
+- Send it to the recruiter that sent you this challenge.
```

Comment:
should we also ask not to include node modules?
we might get back to repos, so hopefully this section will be changed

Reply:
yes, I thought to mention it, but this could be a sign that the candidate doesn't know how npm modules work 😬
I'll add it, thanks!

Reply:
Here are more details on that https://faceit.slack.com/archives/GMSMHQAQJ/p1657551725576399